api/proto/teleport/accessmonitoringrules/v1/access_monitoring_rules.proto

```diff
@@ -0,0 +1,105 @@
+// Copyright 2024 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.accessmonitoringrules.v1;
+
+import "teleport/header/v1/resourceheader.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/accessmonitoringrules/v1;accessmonitoringrulesv1";
+
+// AccessMonitoringRule represents an access monitoring rule resources.
+message AccessMonitoringRule {
+  // header is the header for the resource.
+  teleport.header.v1.ResourceHeader header = 1;
+
+  // kind is a resource kind
+  string kind = 2;
+  // sub_kind is an optional resource sub kind, used in some resources
+  string sub_kind = 3;
+  // version is version
+  string version = 4;
```

Comment:
The Kind/SubKind and Version fields are already part of the `teleport.header.v1.ResourceHeader` type.

Reply:
Will I leave these in then since it the ResourceHeader is being swapped back to Metadata?

Reply:
Have swapped from resourceHeader to metadata as suggested in a different comment so I think these are still needed